### PR TITLE
Add verbose mode and file_name

### PR DESCRIPTION
**Pull request by aarteagamel**
Refactor code
Fix some PEP8 issues

### CODE REVIEW

**Comment by Worvast** [comment]
Could you take advantage of this merge request to upload the dependencies of psutil and Faker to the following versions in requirements.txt :

psutil == 5.7.0
Faker == 4.0.2

Update too in setup.py to fix all dependencies to == version, and check that it still works?
You need sign CLA too, its mandatory

File: devoutils/__version__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 __description__ = 'Devo Python Library.'
 __url__ = 'http://www.devo.com'
-__version__ = "2.0.1"
+__version__ = "2.0.2"
 __author__ = 'Devo'
 __author_email__ = 'support@devo.com'
 __license__ = 'MIT'
-__copyright__ = 'Copyright 2019 Devo'
+__copyright__ = 'Copyright 2020 Devo'
```
Comment (Worvast) [inline devoutils/__version__.py]: In devo we maintain the copyright from the year of the first version release, Gonzalo told me that it is correct to leave it in 2019

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Click==7.0
 requests==2.21.0
 pyyaml>=git+git://github.com/yaml/pyyaml@releases/tag/5.1b3
-devo-sdk==3.0.2
+devo-sdk==3.2.0
```
Comment (Worvast) [inline requirements.txt]: 3.2.1

File: setup.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
     "Programming Language :: Python :: Implementation :: PyPy",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-INSTALL_REQUIRES = ['devo-sdk>=3.0.3,<4', 'faker', 'jinja2>=2.10.1', 'psutil',
+INSTALL_REQUIRES = ['devo-sdk>=3.2.0,<4', 'faker', 'jinja2>=2.10.1', 'psutil',
```
Comment (Worvast) [inline setup.py]: Please update here to 3.2.1 and fix version of all requires like i say in main comments. Thanks.

File: docs/faker.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ config.json file and makes the process interactive.
 If you use ```devo-faker --help``` you can see all the available options.
 Here is the help command result:
 ```
-Usage: devo-faker [OPTIONS]
+Usage: faker_cli.py [OPTIONS]
```
Comment (Worvast) [inline docs/faker.md]: ? Doesn't this continue to be called devo-faker?

File: devoutils/__version__.py (hunk 1 above)
Comment (aarteagamel) [reply devoutils/__version__.py]: Ok I'll revert the change

File: requirements.txt (hunk 2 above)
Comment (aarteagamel) [reply requirements.txt]: This merge will be a major, There are a task pending in the SDK, I'll wait to have the new version of the SDK and update to the new one

File: setup.py (hunk 3 above)
Comment (aarteagamel) [reply setup.py]: Same as above

**Comment by aarteagamel** [comment]
> As it is philosophical, I am not going to complain much, but using "gen" instead of "generator" I do not like because using abbreviations say that it is wrong, being able to make everything very clear, and having autocomplete. But I understand that it can be very long and I let it go.
> 
> It's also not clear to me if modifying a class we should switch to 3.0.0
> I don't know if there are projects that are importing classes that you have modified.

Yes I better to do a Major. Then we can review everything to check if we want other changes to the release. 

I'll change _gen to _generator

**Comment by Worvast** [comment]
I have you changes here: https://github.com/Worvast/python-utils/tree/v3

**Pull request closed without merging**